File: src/main/java/org/pixeltime/enchantmentsenhance/commands/HelpCommand.java
package org.pixeltime.enchantmentsenhance.commands;

import org.bukkit.entity.Player;
import org.pixeltime.enchantmentsenhance.Main;
import org.pixeltime.enchantmentsenhance.manager.SettingsManager;

public class HelpCommand extends SubCommand {

    @Override
    public void onCommand(Player player, String[] args) {
        Main.getMain().commandManager.printHelp(player);
    }


    @Override
    public String name() {
        return "help";
    }


    @Override
    public String info() {
        return "\n&6/enhance help &7- " + SettingsManager.lang.getString(
                "Help.help");
    }


    @Override
    public String[] aliases() {
        return new String[]{"help", "hl", "bangzhu", "bz"};
    }


    @Override
    public String getPermission() {
        return "Enchantmentsenhance.enhance";
    }

}

File: modules/EnchantmentsEnhance-Plugin/src/main/kotlin/org/pixeltime/enchantmentsenhance/command/player/HelpCommand.kt
package org.pixeltime.enchantmentsenhance.command.player

import org.bukkit.entity.Player
import org.pixeltime.enchantmentsenhance.Main
import org.pixeltime.enchantmentsenhance.command.SubCommand

class HelpCommand : SubCommand() {


    override val permission: String
        get() = "Enchantmentsenhance.enhance"

    override fun onCommand(player: Player, args: Array<String>) {
        Main.getCommandManager().printHelp(player)
    }


    override fun name(): String {
        return "help"
    }


    override fun usage(): String {
        return "/enhance help { page }"
    }


    override fun aliases(): Array<String> {
        return arrayOf("help", "hl", "bangzhu", "bz")
    }

}
